Dockerfile

```diff
@@ -40,7 +40,7 @@ RUN rm -rf node_modules log tmp && \
       find /usr/local/bundle/gems -name "*.html" -delete
 
 # Build runtime image
-FROM ruby:3.1.1-alpine3.15 as production
```

Comment:
I think this should reference ruby:3.1-alpine3.15 - ie not specify the patch level - that way we automatically benefit from patch level updates without doing any accidental ruby version bumps